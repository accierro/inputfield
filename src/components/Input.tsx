import * as React from "react";

type InputProps = {};

const Input: React.FC<InputProps> = () => {
  return <div>Have a good coding</div>;
};
export default Input;
