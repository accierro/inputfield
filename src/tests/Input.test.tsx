import * as React from "react";
import { shallow } from "enzyme";
import Input from "../components/Input";
it("Renders without crashing", () => {
  const wrapper = shallow(<Input />);
  expect(wrapper).toMatchSnapshot();
});
