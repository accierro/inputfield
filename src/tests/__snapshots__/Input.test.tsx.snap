// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders without crashing 1`] = `
<div>
  Have a good coding
</div>
`;
